source/usage-examples/struct-tagging.txt

```diff
@@ -0,0 +1,69 @@
+===========================
+Set Custom BSON Field Names
+===========================
+
+.. default-domain:: mongodb
+
+You can customize the way that the Go Driver converts Go
+structs to :manual:`BSON </reference/bson-types/>` using 
+**struct tags**, which are optional instructions you can use with struct
+fields.
+
+The following code declares a ``BlogPost`` struct, where the ``WordCount`` field
+has a struct tag that sets a custom BSON field name of ``word_count``.
+The driver marshals the other fields as the lowercase of the struct
+field name by default:
```

Comment:
Question: Similar to the previous question -- Should this be here if we aren't going too in depth on this page?

Reply:
I think JW suggested I include this